stream_mem_stress_test/memory.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import {
+    KmsKeyringNode,
+    buildClient,
+    CommitmentPolicy
+} from '@aws-crypto/client-node'
+import {AlgorithmSuiteIdentifier} from '@aws-crypto/material-management'
+
+/* This builds client with the REQUIRE_ENCRYPT_REQUIRE_DECRYPT commitment policy.
+ * REQUIRE_ENCRYPT_REQUIRE_DECRYPT enforces that this client only encrypts using committing algorithm suites
+ * and enforces that this client will only decrypt encrypted messages
+ * that were created with a committing algorithm suite.
+ * This is the default if you build `buildClient()`.
+ */
+const { encryptStream, decryptUnsignedMessageStream } = buildClient(
+    CommitmentPolicy.REQUIRE_ENCRYPT_REQUIRE_DECRYPT
+)
+
+import { finished } from 'stream'
+import { createReadStream, createWriteStream } from 'fs'
+import { promisify } from 'util'
```

Comment:
Nit: I could be wrong, but I think we normally put `import`s at the top of the file, and the define our constants.

Reply:
I was following what we do in the node examples:
https://github.com/aws/aws-encryption-sdk-javascript/blob/master/modules/example-node/src/kms_stream.ts#L4-L27